core/templates/services/guppy-configuration.service.spec.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for GuppyConfigurationService
+ */
+
+import { TestBed } from '@angular/core/testing';
+
+import { GuppyConfigurationService } from
+  'services/guppy-configuration.service';
+
+declare global {
+  interface Window {
+    Guppy: Object;
+  }
+}
+
+describe('GuppyConfigurationService', () => {
+  let guppyConfigurationService: GuppyConfigurationService = null;
+  class MockGuppy {
+    static 'remove_global_symbol'(symbol: string): void {}
```

Comment:
@nishantwrp just to check, is this correct? The quotes around the name look a bit weird.

Reply:
BTW, @seanlip when I tried using eslint disable for camelcase, it suggested to wrap the name with quotes.

Reply:
Do you mean 'disable' for snake_case?

Reply:
Actually, using `eslint-disable camelcase` would disable the check for verifying that only camelcase is being used, which is what is required.

Reply:
Yeah, I guess this is right. I enforced this check to avoid using `eslint disable camelcase` so that we have only one convention i.e to wrap camelcase names in `'`.